File: official/docs/java/current/smartrate/retrieve-time-in-transit-statistics.java
package shipments;

import com.easypost.exception.EasyPostException;
import com.easypost.model.SmartRate;
import com.easypost.service.EasyPostClient;

import java.util.List;

public class RetrieveTimeInTransitStatistics {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        List<SmartRate> smartrates = client.shipment.smartRates("shp_...");

        System.out.println(smartrates);
    }
}

File: official/docs/golang/current/smartrate/retrieve-time-in-transit-statistics.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v4"
)

func retrieveTimeInTransitStatistics() {
	client := easypost.New("EASYPOST_API_KEY")

	smartRates, _ := client.GetShipmentSmartrates("shp_...")

	fmt.Println(smartRates)
}
